package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@material-ui/core": "^4.12.1",
+    "@material-ui/core": "^4.12.2",
```

Comment:
why so in this PR? material ui?

Reply:
well, maybe this happend because I got some problems trying to install frist `jest`, but got an error, It seems that `react` already comes with a version of that library, so I had to make some steps in order to fix those problems, and between them was a `npm update`. 
So, what are the steps to fix this?? maybe use a static version of the libraries?